modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -794,6 +799,115 @@ const FDSEntries = ({
 		});
 	};
 
+	const RenameFDSEntryModalContent = ({
```

Comment:
(optional) you could reuse `AddFDSEntryModalContent`, so that label input field is not duplicated. Rename it to something like `SaveFDSEntryModalContent` and implement both logics in it. Would that add that many ifs? If yes, I guess duplication might be acceptable.